src/middlewared/middlewared/plugins/account.py

```diff
@@ -1303,11 +1299,7 @@ async def do_home_copy(self, job, home_old, home_new, username, new_mode, uid):
             })
 
         await perm_job.wait()
-
-        command = f"/bin/cp -a {shlex.quote(home_old) + '/' + '.'} {shlex.quote(home_new + '/')}"
-        do_copy = await run(["/usr/bin/su", "-", username, "-c", command], check=False)
-        if do_copy.returncode != 0:
-            raise CallError(f"Failed to copy homedir [{home_old}] to [{home_new}]: {do_copy.stderr.decode()}")
+        shutil.copytree(home_old, home_new, dirs_exist_ok=True)
```

Comment:
Blocking main event loop with this call here

Reply:
Yeah, I should have switched back to WIP once I realized there were additional issues. Was testing this specifically with CI. I also need to switch user context when performing the action.